app/src/main/kotlin/com/wire/android/notification/FirebaseNotificationManager.kt

```diff
@@ -0,0 +1,59 @@
+package com.wire.android.notification
```

Comment:
Not sure if I am comfortable merging this into the develop, for most of the time this will be a death code, which can be written like in 5 minutes maybe :P Perhaps logging would be sufficent ?